Week1 [STEP 1] 코비(kobe24)

devKobe24 opened the pull request:
@gundy
땀이 나는 첫 PR입니다.

 ## 고민했던 점
 ### 이번 과제에서 다음과 같은 오류들을 만나고 해결 해봤습니다.
- The requested URL returned error: 403
- 원격 저장소 url 변경하기
- git 커밋 되돌리기
- 원격 저장소 리스트 가져오기
- Personal Access Token 설정하

---
 ## 해결이 되지 않은 점
 ### 커밋 되살리기
- 이전 커밋 내용을 길게 썻는데 돌아가는 방법을 몰라 revert라는 명령어를 이용했습니다.
- 이전 커밋 내용을 살린후 그 커밋을 수정하는 방법이 있을까요?

---
## 조언을 얻고 싶은 부분
### 롤백
- 저는 이미 푸쉬한 커밋을 되돌려서 살린 다음 커밋만 수정한 후 다시 푸쉬하고 싶었습니다.
- 이런 상황에서는 revert가 최선이라고 생각해 revert를 사용했습니다.
- 이런 부분에 대해 어떻게 생각하시는지 궁굼합니다.

Commit:
first commit

Commit:
@GUNDY
안녕하세요 건디!! 코비 입니다.
첫 PR을 형식에 맞춰 보냅니다.

- 이번 과제에서 여러 오류와 만나고 해결 하면서 검색과 친해지는 방법을 알게 되었습니다.
	- The requested URL returned error: 403
	- Access Denieded

- touch라는 터미널 커맨드를 검색을 통해 알게되어 txt 파일을 만들어 봤습니다.

- 'git branch -r'을 하면 원격저장소의 브랜치 리스트를 볼 수 있습니다.
- 'git brang -a'을 하면 원격저장소의 모든 브랜치 리스트를 볼 수 있습니다.

- 'git remote -v'로 로컬 저장소가 바라보고 있는 원격 저장소의 url을 확인합니다.
- 'git remore set-url origin {url}' 다음과 같이 로컬 저장소가 바라보는 원격저장소의 url을 바꿀 수 있습니다.

---

- 아직까지는 없습니다.

---

- 현업에서 주로 많이 사용하는 깃 커맨드가 뭐가 있는지 궁궁하고
- 주로 어떤 상황에서 어떤 커맨드를 사용하는지 궁굼합니다.

Commit:
Revert "@GUNDY"

This reverts commit 24880b4c65a69ff4ab6fb0ac28fa2c958a7b9edf.

Commit:
Week1 [STEP 1] 코비(kobe24)

This reverts commit b3899708f1a7395270b06da7b30504c1709d1fdf.

@GUNDY
땀이 나는 첫 PR입니다.

- 이번 과제에서 다음과 같은 오류들을 만나고 해결 해봤습니다.
	- The requested URL returned error: 403
	- 원격 저장소 url 변경하기
	- git 커밋 되돌리기
	- 원격 저장소 리스트 가져오기
	- Personal Access Token 설정하

---

- 이전 커밋 내용을 길게 썻는데 돌아가는 방법을 몰라 revert라는 명령어를 이용했습니다.
- 이전 커밋 내용을 살린후 그 커밋을 수정하는 방법이 있을까요?

---

- 저는 이미 푸쉬한 커밋을 되돌려서 살린 다음 커밋만 수정한 후 다시 푸쉬하고 싶었습니다.
- 이런 상황에서는 revert가 최선이라고 생각해 revert를 사용했습니다.
- 이런 부분에 대해 어떻게 생각하시는지 궁굼합니다.

Commit:
Week1 [STEP 1] 코비(kobe24)
@Gundy93
땀이 나는 첫 PR입니다.

- The requested URL returned error: 403
- 원격 저장소 url 변경하기
- git 커밋 되돌리기
- 원격 저장소 리스트 가져오기
- Personal Access Token 설정하

---

- 이전 커밋 내용을 길게 썻는데 돌아가는 방법을 몰라 revert라는 명령어를 이용했습니다.
- 이전 커밋 내용을 살린후 그 커밋을 수정하는 방법이 있을까요?

---

- 저는 이미 푸쉬한 커밋을 되돌려서 살린 다음 커밋만 수정한 후 다시 푸쉬하고 싶었습니다.
- 이런 상황에서는 revert가 최선이라고 생각해 revert를 사용했습니다.
- 이런 부분에 대해 어떻게 생각하시는지 궁굼합니다.
```

Gundy93 reviewed (commented):
안녕하세요 코비님!

> 이전 커밋 내용을 살린후 그 커밋을 수정하는 방법이 있을까요?

아마도 커밋 메시지를 수정하고자 질문을 주신 것 같습니다! 해당 내용에 대해서는 `git commit amend`를 한 번 검색해보시면 참고가 될 것 같습니다. 
또한 reset과 revert에 대해서 어떤 차이가 있는지도 공부해보시면 도움이 될 것 같습니다!

그리고 현재 PR을 요청하신 브랜치가 main 브랜치인데요, 코비님의 브랜치로 변경해주세요!

아래 코멘트까지 반영되면 바로 머지해드리겠습니다! 😃

Commit:
week1 [STEP 1] Kobe
@Gundy93
다시 한번 더 PR 도전해봅니다 !!

 ## 고민했던 점
 ### 주제
 - The requested URL returned error: 403
 - 원격 저장소 url 변경하기
 - git 커밋 되돌리기
 - 원격 저장소 리스트 가져오기
 - Personal Access Token 설정하기
 - git commit amend
 - reset과 revert의 차이점

 ---

 ## 해결이 되지 않은 점
 ### 주제
 - 터미널 내부에서 `#`을 사용하면 그 뒤에 내용은 무시된다고 써져있는데 PR 예시에는 `## 고민했던점` 이렇게 명시되어있었습니다.
 - 그렇다면 저 '고민했던점' 이라는 단어도 무시되는게 아닌가 궁굼합니다.

 ---

 ## 조언을 얻고 싶은 부분
 ### 주제
 - 어떤 상황에서 commit 내부에 백틱(`)을 사용하는지 궁굼합니다.
 ```
